org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/ZigBeeBindingConstants.java

```diff
@@ -193,6 +198,13 @@
 
     public static final String FIRMWARE_FAILED = "@text/zigbee.firmware.failed";
 
+    // List of channel state constants
+    public static final String STATE_OPTION_BATTERY_MIN_THRESHOLD_REACHED = "minThresholdReached";
+    public static final String STATE_OPTION_BATTERY_THRESHOLD_1_REACHED = "threshold1Reached";
```

Comment:
Same comment as above - do these constants needs to be ```STATE_OPTION_BATTERY_MIN_THRESHOLD_REACHED ``` or can they just be ```STATE_OPTION_BATTERY_MIN_THRESHOLD```?

Reply:
Fine for me, I will change the constant names.